src/jit/decomposelongs.cpp

```diff
@@ -68,21 +68,39 @@ void DecomposeLongs::DecomposeBlock(BasicBlock* block)
     assert(block == m_compiler->compCurBB); // compCurBB must already be set.
     assert(block->isEmpty() || block->IsLIR());
 
-    m_block = block;
+    m_blockWeight = block->getBBWeight(m_compiler);
+    m_range = &LIR::AsRange(block);
+    DecomposeRange();
+}
+
+void DecomposeLongs::DecomposeRange(Compiler* compiler, unsigned blockWeight, LIR::Range& range)
+{
```

Comment:
I would recommend either `DecomposeRangeInternal` or `DecomposeRangeHelper`, as I believe those conventions are used elsewhere.

---
In reply to: [76291726](https://github.com/dotnet/coreclr/pull/6552#discussion_r76291726) [](ancestors = 76291726)